recipe/meta.yaml

```diff
@@ -27,15 +26,13 @@ requirements:
     - capnproto                              # [build_platform != target_platform]
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    # <3 because https://github.com/capnproto/pycapnp/issues/319
-    - cython <3                              # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
```

Comment:
This Cython 3 bug was fixed in PR ( https://github.com/capnproto/pycapnp/pull/320 ), which was included in the 2.0.0 release packaged earlier this year ( https://github.com/conda-forge/pycapnp-feedstock/pull/36 )

So relaxing the Cython pinning

Reply:
Looks like this error still persists. So have dropped this change